api/envoy/config/bootstrap/v2/bootstrap.proto

```diff
@@ -86,6 +86,10 @@ message Bootstrap {
   // within the server.
   ClusterManager cluster_manager = 4;
 
+  // Health discovery service config option.
+  // See api/envoy/api/v2/core/health_check.proto
```

Comment:
You should be able to use RST links to the message proto, e.g. https://github.com/envoyproxy/envoy/blob/master/api/envoy/api/v2/cds.proto#L172. If you don't know the exact path to use there, you can grep for the message in `generated/rst` after a docs run and you should be able to see it.